python/src/scipp/plot/widgets.py

```diff
@@ -89,6 +89,11 @@ def __init__(self,
                 button_style="",
                 layout={"width": "initial"})
 
+            # TODO: hide the profile button for 3D plots. Renable this once
+            # profile picking is supported on 3D plots
+            if ndim == 3:
```

Comment:
I'm not sure this logic belongs in the widget, but maybe you were going to address all this as some later fix?

Reply:
Yes, this is only meant to be temporary.